services/py-api/src/server/handlers/utility_hanlders.py

```diff
@@ -1,7 +1,7 @@
 from fastapi import Response, status
 
 from src.database.db_manager import DB_MANAGER
-from src.server.schemas.response_schemas.schemas import ErrResponse, PongResponse
+from src.server.schemas.response_schemas.schemas import ErrResponse, PongResponse, NumberOfRandomParticipantsResponse
```

Comment:
No this is for testing purposes. Please make sure to clear it out before you commit!

Reply:
Fixed in 1bec98abe406eea6382a5453ac98f3e900d3ddd4